src/converters/light.ts

```diff
@@ -390,6 +460,38 @@ class LightHandler implements ServiceHandler {
     }
     return identifier;
   }
+
+  private handleAdaptiveLighting(value: number): boolean {
+    // Adaptive Lighting active?
+    if (this.adaptiveLighting !== undefined && this.adaptiveLighting.isAdaptiveLightingActive()) {
+      if (this.lastAdaptiveLightingTemperature === undefined) {
+        this.lastAdaptiveLightingTemperature = value;
+      } else {
+        const change = Math.abs(this.lastAdaptiveLightingTemperature - value);
+        if (change < 1) {
+          this.accessory.log.debug(
+            `Adaptive Lighting: Color temperature ${value} skipped for ${this.accessory.displayName}. ` +
+              `Previous: ${this.lastAdaptiveLightingTemperature}`
+          );
```

Comment:
Maybe we should leave this log statement out. I don't think it adds any information, considering the check above.
Have you checked if the the check (ignore if `change < 1` ) is even needed?

Reply:
Ran a short test on my home setup. The check is needed, otherwise it will send the same value every minute or so (depending on how many lights you have in your house that might add quite some _unnecessary_ traffic).

I decided to leave the check and log statement in for now. Might help debug issues in the future.